lib/obsolete.gd

```diff
@@ -756,3 +756,26 @@ DeclareObsoleteSynonym( "IsLexicographicallyLess", "<" );
 ##
 ##  Still used in fr (06/2021)
 DeclareSynonym( "FirstOp", First );
+
+
+#############################################################################
+##
+#A  RadicalGroup( <G> )
+##
+##  <ManSection>
+##  <Attr Name="RadicalGroup" Arg='G'/>
+##
+##  <Description>
+##  is the radical of <A>G</A>, i.e., the largest solvable normal subgroup of <A>G</A>.
+##  <Example><![CDATA[
+##  gap> RadicalGroup(SL(2,5));
+##  <group of 2x2 matrices of size 2 over GF(5)>
+##  gap> Size(last);
+##  2
+##  ]]></Example>
+##  </Description>
+##  </ManSection>
+##
+##  'RadicalGroup' was documented until version 4.11.1.
```

Comment:
It should still be documented! Normally when we rename things and make the old name obsolete, then we add an entry for it (including an index entry) to `doc/ref/obsolete.xml`, and I think that should be done here.

So I'd rather say:
```suggestion
##  'RadicalGroup' was renamed in GAP 4.12
```

Reply:
I can make this change.
But my understanding of the word "documented" is a positive one in the sense that one can rely on documented variables, and using undocumented variables can be risky. This was also the idea behind the (undocumented) function `IsDocumentedWord`; having index entries for deprecated variable names causes that the current implementation of this function does not fit to its intended meaning.

Concerning the sentence "Normally ...":
Currently we do not claim to have a complete list of name changes in the chapter called "Replaced and Removed Command Names" in the Reference Manual. The section about group actions lists "some examples of such name changes", and the section "Miscellaneous Name Changes or Removed Names" states that it lists "some further name changes". I think it would be better to have a complete list, and to state this --a topic for a new issue.